libcontainerd/client_local_windows.go

```diff
@@ -418,6 +418,13 @@ func (c *client) createLinux(id string, spec *specs.Spec, runtimeOptions interfa
 		}
 		configuration.NetworkSharedContainerName = spec.Windows.Network.NetworkSharedContainerName
 	}
+	
+	// Add memory limit (if omittet the memory will be limited to default (1GB) in Hyper-V isolation)
+	if spec.Windows.Resources.Memory != nil {
```

Comment:
These don't need to be nested - the second if won't be evaluated if the first is false.

Reply:
I'm not sure I follow. The first is required in case `specs.Windows.Resources.Memory` is `nil`, right?